Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.StartWith.cs

```diff
@@ -26,6 +26,20 @@ public void When_asserting_string_starts_with_the_same_value_it_should_not_throw
             action.Should().NotThrow();
         }
 
+        [Fact]
+        public void When_expected_string_is_the_same_value_it_should_not_throw()
+        {
+            // Arrange
+            string value = "ABC";
+
+            // Act
+            Action action = () =>
+                value.Should().StartWith(value);
+
+            // Assert
+            action.Should().NotThrow();
```

Comment:
We usually call just the assertion now

Reply:
I copied the structure from another test in the same class. Should I adjust all tests or just this one?

Reply:
I would say yes :)

Reply:
Let's stick to the boyscout rule. As long as a PR makes things better, it's good enough

Reply:
I already adapted the 4 tests in this class :-)